metaworld/envs/mujoco/sawyer_xyz/sawyer_pick_place_v2.py

```diff
@@ -0,0 +1,245 @@
+import numpy as np
```

Comment:
In the module wide docstring of each of these environments, please write a short message describing what the changes are that you have made to these environments.